### PR TITLE
Allow to specify a mountpoint when creating a portstree

### PR DESCRIPTION
**Pull request by smortex**
#### Pull Request (PR) description

This is useful when creating a portstree with the null fetch_method:
this directory is null-mounted in the jail for building ports.

```puppet
  poudriere::portstree { 'default':
    fetch_method => 'null',
    mountpoint   => '/home/romain/FreeBSD/ports',
  }
```

#### This Pull Request (PR) fixes the following issues

n/a

### CODE REVIEW

File: manifests/portstree.pp [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
-  Optional[String[1]]       $branch           = undef,
-  Poudriere::Fetch_method   $fetch_method     = 'svn',
-  Boolean                   $cron_enable      = false,
-  Boolean                   $cron_always_mail = false,
-  Poudriere::Cron_interval  $cron_interval    = { minute => 0, hour => 22, monthday => '*', month => '*', week => '*' },
+  Enum['present', 'absent']      $ensure           = 'present',
+  String[1]                      $portstree        = $name,
+  Optional[String[1]]            $branch           = undef,
+  Poudriere::Fetch_method        $fetch_method     = 'svn',
+  Optional[Stdlib::Absolutepath] $mountpoint       = undef,
```
Comment (ghoneycutt) [inline manifests/portstree.pp:10]: Since this module is not using puppet-strings, the new parameter should be at least documented in the README.md

**Review by ghoneycutt — CHANGES_REQUESTED** [review]

**Comment by ghoneycutt** [comment]
@smortex This looks great! Just needs the new param documented.

File: manifests/portstree.pp (hunk 1 above)
Comment (smortex) [reply manifests/portstree.pp:10]: Hum, yes, I was surprised to not see (at)params comments…  What would it cost to use this?  I mean, may I just add these comments and we are done or do we have to tweak nasty things to finish the work?

Comment (smortex) [reply manifests/portstree.pp:10]: I see non-working options :astonished: (e.g. setting `poudriere::env::makefile` leads to a file with both `source` and `content`)…  I'll add an item in the #52 issue for adding tests which combine all options.

**Comment by smortex** [comment]
@ghoneycutt  We now have inline docs :wink:

File: manifests/portstree.pp (hunk 1 above)
Comment (ghoneycutt) [reply manifests/portstree.pp:18]: Heh, I just meant add the new param to the README, adding puppet-strings for everything is awesome!

**Review by ghoneycutt — APPROVED** [review]

**Pull request merged**